Jint/Runtime/Debugger/BreakPoint.cs

```diff
@@ -1,32 +1,28 @@
 ﻿namespace Jint.Runtime.Debugger
 {
-    public sealed class BreakPoint
+    // BreakPoint is not sealed. It's useful to be able to add additional properties on a derived BreakPoint class (e.g. a breakpoint ID
+    // or breakpoint type) but still let it be managed by Jint's breakpoint collection.
+    public class BreakPoint
```

Comment:
record could also work here

Reply:
For the same reason as above, not sure `record` adds something to `BreakPoint` - value equality, yes, but not entirely sure if value equality is even appropriate for a breakpoint. 🙂 (But that's just my impaired brain speaking)